#Values for status voltages in volts
float64 five_v
float64 twelve_v
float64 twentyeight_v
float64 fourtyeight_v

#Values for status temperatures in F
float64 temp_a
float64 temp_b
float64 temp_c
float64 temp_d
float64 temp_crio
